### ISSUE TITLE
NEW repository contribution, hibernate-cronjob.

### ISSUE THREAD

**Issue by jnpacker** [open-cluster-management-io/community#6]
# Repository Transfer
## What
This is a public repository I would like to contribute to open-cluster-management, that enables a Kubernetes cronjob that will Hibernate and Resume clusters provisioned by Hive.

## Where
[jpacker/hiberanate-cronjob](https://github.com/jnpacker/hibernate-cronjob)

## Requirements
- [x] LICENSE
- [x] CONTRIBUTE.md
- [x] DCO
- [x] SECURITY.md
- [x] OWNERS

## Approvals
- [x] @qiujian16 
- [x] @mikeshng

**Comment by jnpacker** [comment]
Hey folks, looking for a green light.

**Comment by mikeshng** [comment]
it looks good Josh sorry for the late review.

**Comment by qiujian16** [comment]
@jnpacker I think this is an interesting use case, and It would be great to drive a discussion about cluster provisioning in the upcoming community meeting. I think one thing that we could discuss is whether it is valuable to have a more generic cluster provisioning API which is independent on Hive. FYI @pmorie

**Comment by jnpacker** [comment]
So on that note. We started the ClusterCurator for just that purpose. The idea is by using a job to evaluate and provision the ClusterCurator resoruce, we could use Hive and/or other tools to provision a cluster and execute the import commands. It might be worth bringing forward through the community as we delayed it to our next release, but we're starting the proto-types. The plan was to include Go, Python and bash examples of the provisioning job, so that folks could add their own provisioning flows.
